### ISSUE TITLE
Bazel build support

### ISSUE THREAD

**Issue by htuch** [google/libprotobuf-mutator#91]
Would it be possible to upstream the BUILD files for libprotobuf-mutator? We use Bazel in the Envoy project and consume libprotobuf-mutator as a dependency. See https://github.com/envoyproxy/envoy/pull/3139/files#diff-5b5d1ad12e472884fe7b22ec1437193bR1 for a hacked workaround that we're using for now. Would be great to have upstream support for this, in particular since Bazel is a Google technology that we're trying to support more widely.

**Comment by htuch** [comment]
@vitalybuka yes, I don't think my BUILD file is what you want, as you say. We would like our dependencies, in particular the Google authored ones, to have native BUILD files to remove our need to wrap them this way. This seems reasonable from a Google perspective (these BUILD files generally exist internally for Blaze).

**Comment by ctiller** [comment]
We'd love this for gRPC also.

**Comment by sfc-gh-ebrossard** [comment]
Any updates here? It's somewhat surprising to have open-source Google projects still using CMake as the default build system.
